src/components/Links.js

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly ensure you do not disable linters as they help us write cleaner code.